data_dir/lwt_builtin_functions.yaml

```
@@ -0,0 +1,71 @@
+# LWT test: create and update data using LWT.
+### DML ###
+
+# Keyspace Name
+keyspace: lwt_builtin_function_test
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS lwt_builtin_function_test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: blogposts
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE IF NOT EXISTS blogposts (
+        domain uuid,
+        published_date timestamp,
+        lwt_indicator date,
+        url text,
+        author text,
+        title text,
+        PRIMARY KEY((domain, published_date), lwt_indicator)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A table to hold blog posts'
+
+
+### Column Distribution Specifications ###
+
+columnspec:
+  - name: domain
+    population: seq(1..10000000)  #10M possible domains to pick from
+
+  - name: published_date
+    cluster: uniform(1..1001000)         #under each domain we will have max 1000 posts
+
+  - name: lwt_indicator
+    population: seq(1..1001000)
+
+  - name: url
+    size: uniform(30..30)
+
+  - name: title                  #titles shouldn't go beyond 200 chars
+    size: gaussian(10..20)
+
+  - name: author
+    size: uniform(5..20)         #author names should be short
+
+### Batch Ratio Distribution Specifications ###
+
+insert:
+  partitions: fixed(1)            # Our partition key is the domain so only insert one per batch
+
+  select:    fixed(1)/1000        # We have 1000 posts per domain so 1/1000 will allow 1 post per batch
+
+  batchtype: UNLOGGED             # Unlogged batches
+
+
+#
+# A list of queries you wish to run against the schema
+#
+queries:
+   insert_query:
+      cql: insert into blogposts (domain, published_date, lwt_indicator, url, author, title) values (?,?,?,?,?,?) if not exists
```

Should it be like this ?
```suggestion
      cql: insert into blogposts (domain, published_date, lwt_indicator, url, author, title) values (uuid(),?,?,?,?,?) if not exists
```